packages/sms-reminders/helper/get-appointment.ts

```diff
@@ -1,24 +1,164 @@
-import {
-    Appointment,
-    MessageHeader,
-    PatientIdentification,
-    ExternalPatientId,
-    InternalPatientId,
-    PatientName,
-    MotherName,
-    PatientAddress,
-    PhysicalAddress,
-    AppointmentInformation,
-    PlacerAppointmentNumber
+import config from '@amrs-integrations/core';
+import { now } from 'moment';
+import { Appointment } from '../models/appointment';
+
+let CM = config.ConnectionManager.getInstance();
+
+const queryDB = async (person_id: any) => {
+    let amrsCON = await CM.getConnectionAmrs();
+
+    const sql = `
+    select
+	p.person_id,
+	mc.mfl_code as RECEIVING_FACILITY,
+	convert(now(), char) as MESSAGE_DATETIME,
+	pn.given_name as FIRST_NAME,
+	pn.middle_name as MIDDLE_NAME,
+	pn.family_name as LAST_NAME,
+	p.gender as SEX,
+	convert(p.birthdate, char) as DATE_OF_BIRTH,
+	(case when p.birthdate_estimated = 1 then 'Y' else 'N' end) as DATE_OF_BIRTH_PRECISION,
+	convert(p.death_date, char) as DEATH_DATE,
+	concat(p.cause_of_death, '. ', p.cause_of_death_non_coded) as DEATH_INDICATOR,
+	max(case when pa2.person_attribute_type_id = 10 then pa2.value end) as PHONE_NUMBER,
+	pa.city_village as VILLAGE,
+	pa.address1 as POSTAL_ADDRESS,
+	pa.state_province as COUNTY,
+	pa.county_district as SUB_COUNTY,
+	SUBSTRING_INDEX((case when pa2.person_attribute_type_id = 4 then pa2.value end), ' ', 1) as MOTHER_FIRST_NAME,
+	SUBSTRING_INDEX((case when pa2.person_attribute_type_id = 4 then pa2.value end), ' ', -1) as MOTHER_LAST_NAME,
```

Comment:
what will be the output of this? Do they share same attribute type

Reply:
Splitting mother's name into individual first and last names to fit the payload fields.